src/layout/appHeader.tsx

```diff
@@ -1,34 +1,159 @@
-import React, {FC} from 'react'
-import {Link} from 'react-router-dom'
-import {AppBar, Toolbar, Typography, Button} from '@material-ui/core'
+import React, {FC, Dispatch, SetStateAction} from 'react'
+import {useHistory} from 'react-router-dom'
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  Button,
+  makeStyles,
+  IconButton,
+  Menu,
+  MenuItem,
+  useMediaQuery,
+  useTheme,
+  Divider,
+  ListSubheader,
+} from '@material-ui/core'
+import {AccountCircle} from '@material-ui/icons'
+import MenuIcon from '@material-ui/icons/Menu'
 import {useAuth0} from '@auth0/auth0-react'
+import {routes} from '../routes'
 
-export const AppHeader: FC = props => {
-  const {isAuthenticated, loginWithPopup, logout} = useAuth0()
+interface Props {
+  sideDrawOpen: boolean
+  setSideDrawerOpen: Dispatch<SetStateAction<boolean>>
+}
+
+/**
+ * Opening the user menu and side drawer generates a warning in the console
+ * Apparently this is a known issue when running material ui in strict mode https://github.com/mui-org/material-ui/issues/13394
+ * From my reading this will not happen in production and can be resolved by removing strict mode
+ */
+
+export const AppHeader: FC<Props> = ({sideDrawOpen, setSideDrawerOpen}) => {
+  const {isAuthenticated, loginWithPopup, logout, user} = useAuth0()
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null)
```

Comment:
I think we shouldn't use contraction anywhere, as it contributes only to harder readability.

Reply:
Yes, I agree Dave. I'll fix it. Thanks.

Reply:
Great work, Tom! I have just added a couple of comments.

Reply:
I have changed variable/class names in appHeader.tsx, sideDrawer.tsx, and layout.tsx to improve readability.